packages/RatPack.package/RPMockDecoder.class/class/decode..st

```diff
@@ -0,0 +1,6 @@
+as yet unclassified
+decode: anEncodedObject
+
+	anEncodedObject = 'invalid'
+		ifTrue: [^ self error: 'Invalid content']
+		ifFalse: [^ Dictionary newFrom: {'a' -> 5. 'b' -> 7}].
```

Comment:
We should move the dictionary to a class method (see other comments for `testCorrectContentDecoding`).